GNUmakefile

```diff
@@ -23,7 +23,7 @@ test: fmtcheck
 		xargs -t -n4 go test $(TESTARGS) -cover -timeout=30s -parallel=4
 
 testacc: fmtcheck
-	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
+	TF_ACC=1 TF_ACC_MOCK_MCS=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
```

Comment:
i guess this new flag is required for non-acc tests, simple make test (line 20). 
here it will return dummy provider for acceptance tests, but we need normal one
https://github.com/MailRuCloudSolutions/terraform-provider-mcs/pull/45/files#diff-836b967913365db7fcc136323a8a0228e0d4ceeacf3d2ed709650ecfa5742744R81

Reply:
But what about these tests? They seem like acc, and use dummyConfig
(https://github.com/MailRuCloudSolutions/terraform-provider-mcs/blob/d5ee3f457fe5d135b8989f707e296abed95cd73e/mcs/resource_mcs_kubernetes_cluster_test.go#L65)

Reply:
hm, that doesn't look like acc test, cause it's fully mocked and doesn't use real ecosystem

Reply:
rename the test?